src/osemgrep/cli_scan/Scan_subcommand.ml

```diff
@@ -33,13 +33,14 @@ module SS = Set.Make (String)
 (* To run the pro engine, including multistep rules *)
 (*****************************************************************************)
 
-type diff_config = { diff_targets : Fpath.t list; diff_depth : int option }
-
-let default_diff_config = { diff_targets = []; diff_depth = None }
+type diff_config = {
```

Comment:
maybe add a comment like (* see comment in .mli for explanations *)

Reply:
Also it is maybe better to move this type in a separate file; it's becoming an important notion, so maybe it should be in 
a src/core/Differential_scan_config.ml

Reply:
Moved to the separate file.